lib/dep-graph-builders/npm-lock-v2/index.ts

```diff
@@ -190,16 +190,23 @@ const getChildNode = (
   pkgKeysByName: Map<string, string[]>,
   overrides?: Overrides,
 ) => {
+  let version = depInfo.version;
+
   const override =
     overrides &&
-    checkOverrides(
-      [...ancestry, { name, version: depInfo.version }] as Ancestry[],
-      overrides,
-    );
+    checkOverrides([...ancestry, { name, version }] as Ancestry[], overrides);
+
+  if (override) {
+    version = override;
+  }
+
+  if (version.startsWith('npm:')) {
+    version = version.split('@').pop() || version;
```

Comment:
Is `@` guaranteed to always exist in that string?

Reply:
Ok, it returns undefined. Thought it would throw an error.

Reply:
Yes, but if it doesn't you just take the full string anyway

Reply:
Can change to a `lastIndexOf` check

Reply:
Nah, it's OK by me at least.